packages/components/loading/src/loading-provider.tsx

```diff
@@ -474,15 +474,15 @@ const BackgroundComponent = memo(
       position: "fixed",
       right: "md",
       bottom: "md",
```

Comment:
Please refer to `spaces.ts` for reference!
```suggestion
      right: "1rem",
      bottom: "1rem",
```